metacat-main/build.gradle

```diff
@@ -74,14 +75,16 @@ dependencies {
     runtime("io.springfox:springfox-swagger-ui")
     runtime("org.webjars:hal-browser")
     runtime("org.springframework:spring-aspects")
+    runtime("org.apache.logging.log4j:log4j-core")
 
     /*******************************
      * Test Dependencies
      *******************************/
 
     testCompile(project(":metacat-testdata-provider"))
-
     testCompile("io.airlift:testing-mysql-server")
+    testCompile("org.apache.logging.log4j:log4j-core")
```

Comment:
Can we change this to testRuntime?

Reply:
Yes, tried that. The build process will fail using testRuntime
:metacat-main:compileTestGroovy
> Unable to load class org.elasticsearch.node.Node due to missing dependency org/apache/logging/log4j/Logger